recipes/newrelic/apm/.net/linux/net-debian.yml

```diff
@@ -0,0 +1,76 @@
+# Visit our schema definition for additional information on this file format
+# https://github.com/newrelic/open-install-library/blob/main/docs/recipe-spec/recipe-spec.md#schema-definition
+
+name: dotnet-agent-installer
+displayName: .NET Agent Installer
+description:  New Relic install recipe for .NET application on Linux
+repository: https://github.com/newrelic/newrelic-dotnet-agent
+
+installTargets:
+  - type: host
+    os: linux
+    platform: debian
+
+keywords:
+  - Apm
+  - .NET
+  - dotnet
+  - aspnet
+  - core
+
+validationNrql: "SELECT count(*) from Transaction WHERE host like '{{.HOSTNAME}}%' facet entityGuid since 10 minutes ago"
+
+install:
+
+  version: "3"
+  silent: true
+
+  tasks:
+    default:
+      cmds:
+        - task: assert_pre_req
+        - task: remove_any_previous
+        - task: install
+        - task: configure
+
+    assert_pre_req:
+      cmds:
+        - |
+          results=$(dpkg-query -W -f='${Status}\n' dotnet-runtime-*)
+          if [ $? -ne 0 ]; then
+            # Did not find the dotnet-runtime package, don't install the agent
+            exit 1
+          fi
+
+    remove_any_previous:
+      cmds:
+        - sudo apt-get purge -y newrelic-netcore20-agent
+        - |
+          AGENTPROFILE=/etc/profile.d/newrelic-netcore20-agent-path.sh
+          if test -f "$AGENTPROFILE"; then
+              sudo rm $AGENTPROFILE
+          fi
+
+    install:
+      cmds:
+        - echo 'deb http://apt.newrelic.com/debian/ newrelic non-free' | sudo tee /etc/apt/sources.list.d/newrelic.list
+        - wget -O- https://download.newrelic.com/548C16BF.gpg | sudo apt-key add -
```

Comment:
could we use curl instead of wget, we already use curl when we instruct the user to install the CLI, and wget might not be installed

Reply:
This was pulled from our install docs. Sure, I can change that.